test/interceptors/request.test.ts

```diff
@@ -113,12 +113,12 @@ describe('test request interceptor', () => {
   });
 
   it('tests "must revalidate" handling without any headers to do so', async () => {
-    const axios = mockAxios({}, { 'cache-control': 'must-revalidate' });
+    const axios = mockAxios({}, { 'cache-control': 'max-age=1, must-revalidate' });
```

Comment:
Same here, this whole check doesn't make sense, can you recreate it checking if the `must-revalidate` makes `canStale` return false always?
Like one normal request without, a request with `must-revalidate` it and another with `etag` and/or `lastModified` enabled?

Reply:
Said `etag` and/or `lastModified` because they should normally make a storage value go to `stale`.